### ISSUE TITLE
simpleupload.co.uk is down, no CI artifacts accessible

### ISSUE THREAD

**Issue by connorshea** [salesagility/SuiteCRM#8497]
#### Issue

simpleupload.co.uk is down, which means error logs and screenshots from CI jobs can't be accessed.

#### Expected Behavior

The logs and screenshots are uploaded to an accessible location so they can be used by devs to debug test failures.

#### Actual Behavior

They fail to upload because the domain fails to resolve.

From the CI logs:

```
curl -i -F file=@suitecrm.log https://www.simpleupload.co.uk/Api/FileUpload.php?d=upload
curl: (6) Could not resolve host: www.simpleupload.co.uk
```

#### Possible Fix
Either fix simpleupload.co.uk (I think @Dillon-Brown owns it?) or replace simpleupload with another solution.

#### Steps to Reproduce
1. Go to any recent Travis CI job, e.g. https://travis-ci.org/salesagility/SuiteCRM/jobs/642939025
2. Look at the very end of the log, for `curl` commands as part of `push_output.sh`.
3. See that the `curl` commands fail.

**Comment by chris001** [comment]
```
Default server: ns1.digitalocean.com
Address: 2400:cb00:2049:1::adf5:3a33#53
Default server: ns1.digitalocean.com
Address: 173.245.58.51#53
> simpleupload.co.uk
Server:         ns1.digitalocean.com
Address:        2400:cb00:2049:1::adf5:3a33#53

** server can't find simpleupload.co.uk: REFUSED
```
Digital Ocean refused to provide DNS lookup for the domain name. 
Billing/Disk Space issue?
Droplet not turned on?
Fix digital ocean DNS?
Switch to other DNS ie Cloudflare?

**Comment by chris001** [comment]
No plan to turn on `simpleupload.co.uk` ?

**Comment by tsmgeek** [comment]
What about just pushing logs to something like hastebin.com?

```
#!/bin/bash

result=$(curl -sf --data-binary @${1:--} https://hastebin.com/documents) || {
	echo "ERROR: failed to post document" >&2
	exit 1
}

key=$(jq -r .key <<< $result)

echo "https://hastebin.com/${key}"
```

It then returns a url to your log for 30days.

As for screenshots im not sure.

**Comment by chris001** [comment]
Good idea.  Also, screenshots can be uploaded to many free online image upload sites.

**Comment by pgorod** [comment]
In the beginning, there was @Dillon-Brown. He created (if I'm not mistaken) `simpleupload.co.uk`, and he saw that it was good.

Then some expired certificate or domain name or some other evil working blew the whole thing. It has been prophesied that only Dillon can redeem it... we can only raise our hands up to the Highlands and mention him in supplication... @Dillon-Brown! Oh @Dillon-Brown! 🤣

**Comment by tsmgeek** [comment]
https://api.imgbb.com/
